templates/guides/migration-guide-1.0.0.md

```diff
@@ -0,0 +1,79 @@
+---
+page_title: "Fastly Terraform 1.0.0 Migration Guide"
+subcategory: "Migration"
+---
+
+## Fastly Terraform 1.0.0 Migration Guide
+
+**Resources omit `_v1` suffix**:
+
+All resources now have a consistent naming convention which omit the previous v1 suffix.
+
+- `fastly_service_v1` -> `fastly_service_vcl` (renamed with `_vcl` suffix)
+- `fastly_service_acl_entries_v1` -> `fastly_service_acl_entries`
+- `fastly_service_dictionary_items_v1` -> `fastly_service_dictionary_items`
+- `fastly_service_dynamic_snippet_content_v1` -> `fastly_service_dynamic_snippet_content`
+- `fastly_user_v1` -> `fastly_user`
+
+When switching to the new 1.0.0 release, some existing projects might see an error such as:
```

Comment:
General point - consider omitting the version number when referring to it as an upgrade - as described above the guide probably applies beyond the 1.0.0 release.  Or say "Starting from 1.0.0..."

```suggestion
After upgrading, some existing projects might see an error such as:
```